models/Customers.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const db = require('../db/index.js');
+mongoose.Promise = global.Promise;
+
+let CustomersSchema = mongoose.Schema({
+  "customerId":  { type: Number, min: 1, max: 50  },
+  "productId":  { type: Number, min: 1, max: 100  },
```

Comment:
As I mentioned in generate_customers.js, shouldn't productId and purchased be arrays that match up with the reviews?